packages/components/src/NavTree/NavTree.tsx

```diff
@@ -160,18 +180,20 @@ const NavTreeLayout = ({
     ...(isIndicatorToggleOnly ? indicatorToggleOnlyProps : undefined),
   })
 
+  const handleContentBlur = useWrapEvent((event: FocusEvent<HTMLElement>) => {
+    if (!isIndicatorToggleOnly && onBlurDisclosureToggle)
+      onBlurDisclosureToggle(event)
+  }, onContentBlur)
   const handleContentClick = useWrapEvent((event: MouseEvent<HTMLElement>) => {
```

Comment:
Nit, add white-space here to make it easier to discern two different functions :)

```suggestion
  }, onContentBlur)
  
  const handleContentClick = useWrapEvent((event: MouseEvent<HTMLElement>) => {
```